src/neighbors.cc

```diff
@@ -959,7 +963,7 @@ ignoreMulticastReply(CachePeer * p, MemObject * mem)
     if (!p->options.mcast_responder)
         return 0;
 
-    if (peerHTTPOkay(p, mem->request))
+    if (peerHTTPOkay(p, mem->request, nullptr))
```

Comment:
This is the only context where ps_state proved to be unavailable.

Reply:
```suggestion
    if (peerHTTPOkay(p, mem->request, /* XXX: missing ps */ nullptr))
```